banyand/query/processor_topn.go

```diff
@@ -362,13 +362,8 @@ func (naggr *postNonAggregationProcessor) Val(tagNames []string) []*measurev1.To
 		})
 	}
 
-	slices.SortStableFunc(topNLists, func(a, b *measurev1.TopNList) bool {
-		if a.GetTimestamp().GetSeconds() < b.GetTimestamp().GetSeconds() {
-			return true
-		} else if a.GetTimestamp().GetSeconds() == b.GetTimestamp().GetSeconds() {
-			return a.GetTimestamp().GetNanos() < b.GetTimestamp().GetNanos()
-		}
-		return false
+	slices.SortStableFunc(topNLists, func(a, b *measurev1.TopNList) int {
+		return int(a.GetTimestamp().GetSeconds() - b.GetTimestamp().GetSeconds())
```

Comment:
@lujiajing1126 Please review this update. I need to modify it due to a breaking change to the function which is from `golang.org/x/exp` .